web/src/components/RichText.js

```diff
@@ -53,10 +53,19 @@ const setupTinyMCE = upload => editor => {
   });
 };
 
-const uploadImage = upload => (blob, success, failure) =>
+const uploadImage = (upload, domID, onSync) => (blob, success, failure) =>
   upload(blob.blob())
     .then(url => {
       success(url);
+      /* global tinyMCE */
+      // ☝️ lets eslint know that tinyMCE is defined. Once we've told Tiny the
+      // URL of the uploaded image, it'll update the content of the textbox,
+      // but it does NOT call the change event. That means that dropped images
+      // aren't saved as IMG tags unless the user makes additional changes to
+      // the textbox content. To get around that, we can get a reference to
+      // the editor, pull the content, and manually trigger the event.
```

Comment:
![it's all we can do](https://media0.giphy.com/media/3og0IJgyj4hijAEyk0/giphy.gif)